src/GeneralStateTestsFiller/stZeroKnowledge/ecpairing_one_point_on_curve_not_in_subgroupFiller.json

```diff
@@ -0,0 +1,288 @@
+                "code": "0x"
+            },
+            "90f0b1ebbba1c1936aff7aaf20a7878ff9e04b6c": {
+                "storage": {},
+                "nonce": "0",
+                "balance": "1000000000000000000",
+                "code": "0x"
+            }
+        },
+        "transaction": {
+            "data": [
+                "0x30c8d1da000000000000000000000000000000000000000000000000000000000000002000000000000000000000000000000000000000000000000000000000000000c0000000000000000000000000000000000000000000000000000000000000000100000000000000000000000000000000000000000000000000000000000000021382cd45e5674247f9c900b5c6f6cabbc189c2fabe2df0bf5acd84c97818f5081246178655ab8f2f26956b189894b7eb93cd4215b9937e7969e44305f80f521e08331c0a261a74e7e75db1232956663cbc88110f726159c5cba1857ecd03fa641fbf8045ce3e79b5cde4112d38bcd0efbdb1295d2eefdf58151ae309d7ded7db000000000000000000000000000000000000000000000000000000000000000130644e72e131a029b85045b68181585d97816a916871ca8d3c208c16d87cfd451382cd45e5674247f9c900b5c6f6cabbc189c2fabe2df0bf5acd84c97818f5081246178655ab8f2f26956b189894b7eb93cd4215b9937e7969e44305f80f521e08331c0a261a74e7e75db1232956663cbc88110f726159c5cba1857ecd03fa641fbf8045ce3e79b5cde4112d38bcd0efbdb1295d2eefdf58151ae309d7ded7db"
+            ],
+            "gasLimit": [
+                "0x39be8", "90000", "110000", "200000"
+            ],
+            "gasPrice": "0x1",
+            "nonce": "0x14",
+            "secretKey": "0x044852b2a670ade5407e78fb2863c51de9fcb96542a07186fe3aeda6bb8a116d",
+            "to": "0xc305c901078781c232a2a521c2af7980f8385ee9",
+            "value": [
+                "0x"
+            ]
+        },
+        "expect": [
+            {
+                "indexes": {
+                    "data": 0,
+                    "gas": -1,
+                    "value": 0
+                },
+                "network": [
+                    ">=Byzantium"
+                ],
+                "result": {
+                    "24143873e0e0815fdcbcffdbe09c979cbf9ad013": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "0000000000000000000000000000000000000001": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1",
+                        "code": "0x"
+                    },
+                    "dceceaf3fc5c0a63d195d69b1a90011b7b19650d": {
+                        "storage": {},
+                        "nonce": "0",
+                        "balance": "1000000000000000000",
+                        "code": "0x"
+                    },
+                    "c305c901078781c232a2a521c2af7980f8385ee9": {
+                        "storage": {
+                            "0x00": "0x290decd9548b62a8d60345a988386fc84ba6bc95484008f6362f93160ef3e563"
```

Comment:
nothing changed

Reply:
Correct, this is a failure test, just like https://github.com/ethereum/tests/blob/develop/src/GeneralStateTestsFiller/stZeroKnowledge/ecpairing_one_point_not_in_subgroupFiller.json, except the point are on the G2 curve.  Previous tests were failing but failing because of the not on curve test.